src/guacamole.yml

```diff
@@ -30,7 +32,6 @@
   # boto3 is needed during deployment to fetch SSL certificates.
   # pystache may be used during deployment as part of the process that
```

Comment:
I think this comment needs to be indented more.

Reply:
The comment begins at column 3 in line with `tasks` which is the same as in the current file in develop

Reply:
Right, but the `tasks:` line below the comment has been removed, so the comment instead should apply to the `ansible.builtin.pip` task.

Reply:
I see what you're saying.  See 4aab2fe

Reply:
Thanks, that's perfect.  Sorry if I was unclear earlier; I try not to be but often fail.